src/app/screens/Keysend/index.tsx

```diff
@@ -107,10 +131,23 @@ function Keysend() {
           </Container>
         </>
       ) : (
-        <Container maxWidth="sm">
-          {destination && <PublisherCard title={destination} />}
+        <Container justifyBetween maxWidth="sm">
+          <ResultCard
+            isSuccess
+            message={
+              !destination
+                ? successMessage
+                : `${amount} SATS ${
+                    showFiat ? `(${fiatAmount})` : ``
+                  } ${tCommon("were_sent_to")} ${destination}`
```

Comment:
shouldn't this full sentence be in i18n with placeholders?

it might not fit in other languages.

Reply:
Done!